src/components/radio-button/radio-button-panel/radio-button-panel.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { assert, expect } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+
+import { fixture, testA11yTreeSnapshot } from '../../core/testing/private/index.js';
+
+import { SbbRadioButtonPanelElement } from './radio-button-panel.js';
+
+describe('sbb-radio-button-panel', () => {
+  let element: SbbRadioButtonPanelElement;
+
+  describe('should render unchecked', async () => {
+    beforeEach(async () => {
+      element = (await fixture(
+        html`<sbb-radio-button-panel value="radio-value">
+          Label
+          <span slot="subtext">Subtext</span>
+          <span slot="suffix">Suffix</span>
+        </sbb-radio-button-panel>`,
+      )) as SbbRadioButtonPanelElement;
+      assert.instanceOf(element, SbbRadioButtonPanelElement);
+    });
+
+    it('DOM', async () => {
+      await expect(element).dom.to.be.equalSnapshot();
+    });
+
+    it('Shadow DOM', async () => {
+      await expect(element).shadowDom.to.be.equalSnapshot();
+    });
+  });
+
+  beforeEach(async () => {
+    element = await fixture(
+      html`<sbb-radio-button-panel value="radio-value">
+        Label
+        <span slot="subtext">Subtext</span>
+        <span slot="suffix">Suffix</span>
+      </sbb-radio-button-panel>`,
+    );
+  });
```

Comment:
Do i miss something? what's the difference with previous case?

Reply:
Messed up while copy-pasting, this should've been the checked case. Fixing it